packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -728,10 +771,12 @@ export function makeVirtualObjectManager(
         const context = { state };
         // `context` does not need a linkToCohort because it holds the facets (which hold the cohort)
         unweakable.add(context);
-        context.self = bindMethods(context, behaviorTemplate);
-        toHold = Far(tag, context.self);
-        linkToCohort.set(Object.getPrototypeOf(toHold), toHold);
-        unweakable.add(Object.getPrototypeOf(toHold));
+        const self = harden({ __proto__: prototypeTemplate });
+        context.self = self;
+        contextMapTemplate.set(self, context);
+        toHold = self;
+        // linkToCohort.set(Object.getPrototypeOf(toHold), toHold);
+        // unweakable.add(Object.getPrototypeOf(toHold));
```

Comment:
If this code is to remain as a comment, it would be good to have an explanation as to why. Is it the old way? the approach to aspire to?

Reply:
Leftover from intermediate state. Gone.